lib/rules/heuristic/group.js

```diff
@@ -1,45 +0,0 @@
-
```

Comment:
That rule cannot be removed. We need to list the detected deliverers on the interface so the user can check if everything is ok